pkg/tesla/telemetry/codegen/codegen.go

```diff
@@ -160,41 +192,45 @@ func Generate(packageName, outerOutputPath, innerOutputPath string) error {
 	return nil
 }
 
-func assembleTeslaTypeInformation() (map[string]TeslaTypeDescription, error) {
-	out := make(map[string]TeslaTypeDescription)
+func assembleTeslaTypeInformation() (map[string]TeslaGoValueType, error) {
+	out := make(map[string]TeslaGoValueType)
 
+	// This is protobuf reflection. The protos.Value type in Go does
+	// not have all of these fields. Maybe we should reflect in Go,
```

Comment:
I think the proto docs discourage using go reflect since it could break with proto updates

Reply:
My desire here was for the config file to only reference the .proto file, which leads you to this place. I wonder if it's possible to do that while jettisoning protoreflect.

I think let's roll with this for now and revisit.